victorLi/assignments/treesBFS/lc637/lc637.js

```diff
@@ -0,0 +1,67 @@
+// Problem Statement #
+
+// Given a binary tree, populate an array to represent the averages of all of its levels.
+
+
+/*
+
```

Comment:
Would recommend being a little more detailed in your overview. Don't forget to consider edge cases well. I would also recommend including a note about the time and space complexity of your solution. Additional test cases could be helpful as well.